.github/workflows/full-ci.yml

```diff
@@ -75,9 +75,8 @@ jobs:
   ci6:
     uses: ./.github/workflows/ci-linux_mac.yml
     with:
-      ci_option: Experimental
       matrix_image: ubuntu-22.04
-      bootstrap_args: '--enable=experimental-features,serialization --enable-release-symbols'
+      bootstrap_args: 'serialization --enable-release-symbols'
```

Comment:
Change `serialization` to `--enable-serialization`, and add a `ci_option: Serialization`?

Reply:
Done.